src/sidebar/components/SelectionTabs.tsx

```diff
@@ -50,21 +56,20 @@ function Tab({
 
   return (
     <LinkButton
-      classes={classnames(
-        'inline bg-transparent min-w-[5.25rem] text-color-text hover:!no-underline',
-        {
-          'font-bold': isSelected,
-        }
-      )}
+      classes={classnames('!inline bg-transparent min-w-[5.25rem]', {
```

Comment:
What is the `!inline` for?

Reply:
Thanks for spotting this as it is a double-derp:

1. An [issue (since fixed)](https://github.com/hypothesis/frontend-shared/issues/650) was preventing the `inline` prop from being respected on `LinkButton`, which required the `!inline` workaround.
2. However, even so, this `LinkButton` doesn't need to _be_ inline, as it is a child of a `flex` (row) container. I wonder if I added a `flex` parent at some point recently and forgot that I did so and forgot to remove the `inline` for these `LinkButton`s.

Anyway, sloppy. I've removed all references to inline-y-ness for these.